frontend/discourse/app/components/reviewable-refresh/ip-lookup.gjs

```diff
@@ -19,11 +42,186 @@ export default class ReviewableIpLookup extends Component {
+    });
+  }
+
   <template>
     {{#if this.showIpLookup}}
-      <div class="reviewable-ip-lookup">
-        <this.IpLookupComponent @ip="adminLookup" @userId={{this.target}} />
-      </div>
+      {{#if this.loading}}
+        <ConditionalLoadingSpinner @size="small" @condition={{this.loading}} />
+      {{else if this.location}}
+        <div class="reviewable-ip-lookup">
+          <div class="review-insight__item">
+            <div class="review-insight__content">
+              <div class="review-insight__label">
+                {{i18n "ip_lookup.title"}}
+                <span class="ip-lookup-powered-by">
+                  ({{htmlSafe (i18n "ip_lookup.powered_by")}})
+                </span>
+              </div>
+              <div class="review-insight__description">
+                {{#if this.location.hostname}}
+                  {{i18n "ip_lookup.hostname"}}:
+                  {{this.location.hostname}},
+                {{/if}}
+                {{i18n "ip_lookup.location"}}:
+                {{#if this.location.location}}
+                  <a
+                    href="https://maps.google.com/maps?q={{this.location.latitude}},{{this.location.longitude}}"
+                    rel="noopener noreferrer"
+                    target="_blank"
+                  >
+                    {{this.location.location}}
+                  </a>
+                {{else}}
+                  {{i18n "ip_lookup.location_not_found"}}
+                {{/if}}
+                {{#if this.location.organization}}
+                  ,
+                  {{i18n "ip_lookup.organisation"}}:
+                  {{this.location.organization}}
+                {{/if}}
+              </div>
+              {{#if this.totalOthersWithSameIP}}
+                <div class="review-insight__description">
+                  <button
+                    type="button"
+                    {{on "click" this.showOtherAccountsModal}}
+                    class="btn-link ip-lookup-other-accounts-link"
+                  >
+                    {{i18n
+                      "ip_lookup.other_accounts_with_ip"
+                      count=this.totalOthersWithSameIP
+                    }}
+                  </button>
+                </div>
+              {{/if}}
+            </div>
+          </div>
+        </div>
+      {{/if}}
     {{/if}}
   </template>
 }
+
+const OtherAccountsModal = <template>
+  <DModal
+    @title={{i18n
+      "ip_lookup.other_accounts_with_ip"
+      count=@model.totalOthersWithSameIP
+    }}
+    @closeModal={{@closeModal}}
+    class="ip-lookup-other-accounts-modal"
+  >
+    <:body>
+      {{#if @model.otherAccountsLoading}}
+        <ConditionalLoadingSpinner
+          @size="small"
+          @condition={{@model.otherAccountsLoading}}
+        />
+      {{else if @model.otherAccounts}}
+        <IpLookupAccountsTable @accounts={{@model.otherAccounts}} />
+      {{/if}}
+    </:body>
+    <:footer>
+      {{#if @model.totalOthersWithSameIP}}
```

Comment:
Maybe we should also check here for `@model.otherAccountsLoading`, just in case it takes longer and the admin starts clicking button